.travis.yml

```diff
@@ -30,7 +35,13 @@ cache:
   - "$HOME/.composer/cache"
 
 before_install:
-- phpenv config-rm xdebug.ini
+- >
+  if [[ $COVERAGE = yes ]]; then
+    export PHPUNIT_OPTIONS="--coverage-clover=/tmp/coverage.clover";
```

Comment:
Please move the coverage file location to a variable so we define it only in one place and can use it here and in the upload line.

Reply:
I wish you tell that earlier ;-)